arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/gas/projector/GasProjector.java

```diff
@@ -132,31 +138,32 @@ public GasProjection of(MessageFrame frame, OpCode opCode) {
           PC,
```

Comment:
Side note: we will also have to deal with the different opcodes (new ones and renamed ones.)